airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3553,7 +3553,7 @@ components:
           description: The dataset update time
           nullable: false
 
-    DatasetDagRef:
+    DagDatasetConsumer:
```

Comment:
Same as other PR, wdyt about `DatasetConsumer` or `DatasetConsumerDag` instead?

Reply:
+1 `DatasetConsumerDag`

Reply:
+1 DatasetConsumerDag TOO

Reply:
well, i guess the people have spoken

Reply:
i'll just mention though... the reason i thought it makes sense to put Dag first, is cus it's the reference from dag to dataset.  

so if we didn't have to specify type of reference (i.e. consumer vs producer) then it would be a pretty easy choice i think.... we'd just call it `dag_dataset` since it's just a mapping from dag to dataset.

and so i figured, just add a suffix indicating type: dag_dataset_consumer

but, i guess another way to think about it is, not so much as a mapping table, but more as an entity unto itself, e.g. a DatasetConsumer entity of type Dag....  which... i guess is how y'all are thinking about it.

Reply:
Like the other one, maybe `DatasetConsumingDag` instead?

Reply:
actually had other thought please see thread https://github.com/apache/airflow/pull/25919#discussion_r955522680